.github/workflows/ci.yml

```diff
@@ -65,8 +65,10 @@ jobs:
         fetch-depth: 0
     - name: Build Debian package
       run: |
+        set -x
         git fetch --recurse-submodules=no https://github.com/linuxcnc/linuxcnc refs/tags/*:refs/tags/*
         ./scripts/travis-install-build-deps.sh
         debuild -uc -us -I -i -j$(nproc)
-        sudo apt install -y lintian
+        sudo apt-get install ../*.deb
```

Comment:
That's correct.  The way the build system is set up, with the "RIP" build distinct from a system install, it's not possible to run tests from within `debian/rules`.  It would be possible to test uspace threads (but not kthreads) for a system install only with a build system overhaul.